app/frontend/src/components/forms/FormSubmission.vue

```diff
@@ -20,22 +35,7 @@
               <span>View All Submissions</span>
             </v-tooltip>
           </span>
-          <DeleteSubmission @deleted="onDelete" :submissionId="submissionId" />
-        </v-col>
-        <!-- page title -->
-        <v-col cols="12">
-          <h1>{{ form.name }}</h1>
-          <p>
-            <strong>Submitted:</strong>
-            {{ formSubmission.createdAt | formatDateLong }}
-            <br />
-            <strong>Confirmation ID:</strong>
-            {{ formSubmission.confirmationId }}
-            <br />
-            <strong>Submitted By:</strong>
-            {{ formSubmission.createdBy }}
-            <br />
-          </p>
+          <DeleteSubmission class="d-print-none" @deleted="onDelete" :submissionId="submissionId" />
```

Comment:
Does this need d-print class? It's on the containing column so the whole div containing all this would hide right?

Reply:
Good catch - yea that class may not be necessary at all due to inheritance.

Reply:
good catch, removing this class on the component in new commit.